src/plots/attributes.js

```diff
@@ -167,6 +167,7 @@ module.exports = {
         _isLinkedToArray: 'transform',
         editType: 'calc',
         description: [
+            'This attribute is deprecated and it may be removed from the API in next major version.',
```

Comment:
```suggestion
            'WARNING: All transforms are deprecated and may be removed from the API in next major version.',
```

Just trying to make it a little more alarming (and the scope clearer) to anyone who finds this.